woocommerce/woocommerce-gutenberg-products-block
# Sync email from customer object to cart response

On the recent refactor, we forget to fetch the customer email when making a response.

### Testing

1. In Checkout, fill an email, let it sync
2. Fill first name, let it sync.
3. Select a shipping method, email should persist.
